doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,137 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run VxWorks as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
```

Comment:
Can you add a link to the VxWorks product page here: https://www.windriver.com/products/vxworks/?

@dbkinder we probably need to add the copyright sign somewhere (at least once)?

You can add this line at the bottom of your file:
```
.. _VxWorks: https://www.windriver.com/products/vxworks/
```

And then re-use that link using
```
`VxWorks`_
```
throughout the doc

Reply:
Thanks for your comment. Will do.